tests/unit/versionUpdaterTest.js

```diff
@@ -0,0 +1,96 @@
+const {describe} = require('jest-circus');
```

Comment:
not sure this is required, in our other tests we don't `import` or `require` `describe`, I think it's a keyword in `jest`

Reply:
I think it was an auto import, I can remove it

Reply:
Yes, jest automatically find this dependency, I use auto importer (a real life saver!) but sometimes it misbehaves... sorry